### ISSUE TITLE
What is the normal loss range

### ISSUE THREAD

**Issue by taicaile** [AlexeyAB/darknet#5808]
Hi,

I tried to train a model which detect vehicles only, and I get the following loss chart, I want to know what is the normal range of the loss?

Where I can see the yolov4 training loss history chart?

![image](https://user-images.githubusercontent.com/23215696/83418389-17b37100-a456-11ea-9d57-0bf2b45c3535.png)

Thanks.
